### PR TITLE
Custom paging

### PR DESCRIPTION
**Pull request by JakobInterad**

### CODE REVIEW

File: GridMvc.Site/Models/Grids/OrdersAjaxPagingGrid.cs [hunk 1]
```diff
@@ -27,6 +27,7 @@ public AjaxGridPager(IGrid grid)
         public int CurrentPage { get; set; }
 
         public int? ItemsCountOverwrite { get; set; }
+        public bool? CustomPaging { get; set; }
```
Comment (DavidInterad) [inline GridMvc.Site/Models/Grids/OrdersAjaxPagingGrid.cs:30]: What is the difference in behavior of `false` and `null`? Could this be non-nullable?

File: GridMvc/Html/IGridHtmlOptions.cs [hunk 2]
```diff
@@ -104,6 +104,12 @@ public interface IGridHtmlOptions<T> : IHtmlString
         /// </summary>
         IGridHtmlOptions<T> WithGridItemsCount(string formatString = null);
 
+        /// <summary>
+        ///     Disable default paging
```
Comment (DavidInterad) [inline GridMvc/Html/IGridHtmlOptions.cs:108]: This should be described a bit more in detail.
What does disabling default paging mean, what are the behavior changes/consequences?

It is not obvious from this description, that the paging navigation would be displayed nonetheless, but all items of the `IQueryable` will be displayed without restricting them to the current page.

File: GridMvc/Pagination/IGridPager.cs [hunk 3]
```diff
@@ -24,6 +24,11 @@ public interface IGridPager
         /// </summary>
         int? ItemsCountOverwrite { get; set; }
 
+        /// <summary>
+        ///     Disables default paging if set to true
```
Comment (DavidInterad) [inline GridMvc/Pagination/IGridPager.cs:28]: What does it mean to disable default paging?

**Review by DavidInterad — CHANGES_REQUESTED** [review]

File: GridMvc.Site/Models/Grids/OrdersAjaxPagingGrid.cs (hunk 1 above)
Comment (JakobInterad) [reply GridMvc.Site/Models/Grids/OrdersAjaxPagingGrid.cs]: Is now a non-nullable

File: GridMvc/Pagination/IGridPager.cs (hunk 3 above)
Comment (JakobInterad) [reply GridMvc/Pagination/IGridPager.cs:28]: Is now described better

**Review by DavidInterad — APPROVED** [review]

**Pull request merged**